anyio/__init__.py

```diff
@@ -501,7 +501,8 @@ async def create_unix_server(
 
 async def create_udp_socket(
     *, interface: Optional[IPAddressType] = None, port: Optional[int] = None,
-    target_host: Optional[IPAddressType] = None, target_port: Optional[int] = None
+    target_host: Optional[IPAddressType] = None, target_port: Optional[int] = None,
+    address_family: Optional[int] = socket.AF_UNSPEC
```

Comment:
Add this parameter to the docstring please. Emphasize it's only necessary if neither ìnterface` or `target_host` have been specified.